### PR TITLE
Remove unneeded variables and conditions

### PR DESCRIPTION
**Pull request by dcbriccetti**
I’ve recently discovered these helpful examples. I’d like to offer some code simplifications.

### Accelerometer
If I’m not mistaken, x, y, and z are never (or **extremely** rarely) 0, so the “if”s are not needed, and then the R, G, B variables are not needed. Did I miss something?

### ~~Buttons~~
~~I find my new Python students are comfortable with conditional expressions, but if the
consensus here is that the if statement is preferred, I can remove that commit.~~

### CODE REVIEW

**Comment by dcbriccetti** [comment]
Hi Scott. Thanks for having a look.

I removed the if expression commit from the branch.

About the 9 lines of code (with three “if”s) to possibly ignore unreasonable values, how likely is that? Perhaps I’ll go follow the code and see what `cpx.acceleration` can return.

**Comment by dcbriccetti** [comment]
You know, I probably should have started by asking @kattni the reason for the “if”s.

**Comment by tannewt** [comment]
I can't remember why they are there. Hopefully she does. If not, we can always try it and see if it breaks. :-)

**Comment by dcbriccetti** [comment]
If the ```if```s *are* needed, an explanatory comment might be in order.

**Comment by dcbriccetti** [comment]
Thanks, @kattni. 

What’s best to do here? Would you prefer to leave the file as it is?

**Comment by kattni** [comment]
Is it worth maintaining that functionality by retaining the `R = 0` etc. and then changing this line to:
```
cpx.pixels.fill(((R + abs(int(x))), (G + abs(int(y))), (B + abs(int(z)))))
```
Turns into a lot of parens, but allows for increasing the overall brightness of things by allowing increased baseline RGB values.

Or does setting `R = x + 0` and then making it `cpx.pixels.fill((abs(int(R)), ...` work? That would simplify the `fill` statement, and also might make it easier to see the RGB/xyz relationship. 

I haven't tested either of those, I'm simply suggesting here.

@dcbriccetti What do you think?

**Comment by dcbriccetti** [comment]
I think there’s value in explicitly showing the mapping between acceleration components and RGB components.

File: examples/circuitplayground_acceleration_neopixels.py [hunk 1]
```diff
@@ -13,15 +15,17 @@
...
-        x, y, z = cpx.acceleration
-        print((x, y, z))
-        if x:
-            R = R + abs(int(x))
-        if y:
-            G = G + abs(int(y))
-        if z:
-            B = B + abs(int(z))
-        cpx.pixels.fill((R, G, B))
+        def color_amount(accel_component):
```
Comment (kattni) [inline examples/circuitplayground_acceleration_neopixels.py]: We typically include `def`s before the loop at the top of the code below the `import`s. Consider moving this.

**Comment by dcbriccetti** [comment]
Where’s the embarrassed emoji? :-)

Yes, I sensed this was going the wrong direction.

**Comment by dcbriccetti** [comment]
>retaining the R = 0 etc. ...

OK, I like that idea.

**Comment by dcbriccetti** [comment]
Glad to be working with you to help with education.

Do you thing there’s much value to the Adafruit audience of my more complicated version? I can always just use it with my students, and maybe make a YouTube video explaining it.

**Comment by kattni** [comment]
@dcbriccetti I think there's value to having multiple options. I would suggest explaining it thoroughly though, as it was unclear to me, so I imagine it will be unclear to many. If you make a video explaining it, you could include a link in a comment.

**Comment by dcbriccetti** [comment]
The video is here, https://youtu.be/eNpPLbYx-iA, and I’ll make a new pull request for the new program.

**Pull request merged**